src/main/java/pl/javastart/task/Main.java

```diff
@@ -3,6 +3,27 @@
 public class Main {
 
     public static void main(String[] args) {
+        Vehicle car1 = new Car("Toyota RAV4", 60, 7, false);
```

Comment:
Jeżeli stworzysz tablicę typu `Car[]`, a nie `Vehicle[]`, to w Vehicle nie będziesz musiał mieć tych pustych metod związanych z klimatyzacją.
Polimorfizm należy pamiętać jako "używamy najogólniejszego typu, który nie powoduje utraty funkcjonalności". W naszym przypadku klimatyzacja pojawia się dopiero w klasie Car, więc to najogólniejszy typ, którego powinniśmy tutaj używać.